src/test/java/org/redischool/attendance/FeatureTestForSecurity.java

```diff
@@ -17,98 +18,99 @@
-//        }
-//)
-//@RunWith(SpringJUnit4ClassRunner.class)
-//@ActiveProfiles("cloud")
-//public class FeatureTestForSecurity extends FeatureTestScaffolding {
-//
-//    // redi-project-tenjin@googlegroups.com has access to Chasing Unicorns but not to Android App. Configured in okta admin console with group settings
-//    private static final String COURSE_NAME_I_CANNOT_ACCESS = "Android App";
-//
-//    private static WebDriver driver;
-//
-//    @Value("${okta.test.username}")
-//    private String username;
-//
-//    @Value("${okta.test.password}")
-//    private String password;
-//
-//    @Before
-//    public void setUp() {
-//        boolean firstRun = driver == null;
-//
-//        if (firstRun) {
-//            driver = createWebDriver();
-//        }
-//
-//        initFluent(driver);
-//        setBaseUrl("http://localhost:" + port + "/");
-//    }
-//
-//    @AfterClass
-//    public static void tearDown() {
-//        if (driver != null) {
-//            driver.quit();
-//        }
-//    }
-//
-//    @Test
-//    public void testSeesOnlyAssignedCourses() {
-//        // ACT
-//        goHome();
-//
-//        // ASSERT
-//        assertThat(isCourseVisible(COURSE_NAME)).isTrue();
-//        assertThat(isCourseVisible(COURSE_NAME_I_CANNOT_ACCESS)).isFalse();
-//    }
-//
-//    @Test
-//    public void testHasAccessOnlyToCoursesAssigned() {
-//        // ARRANGE
-//        goHome();
-//
-//        // ACT
-//        selectCourse(COURSE_NAME);
-//
-//        // ASSERT
-//        assertThat($("h1#courseName").first().text()).isEqualTo(COURSE_NAME);
-//    }
-//
-//    @Test
-//    public void testHasNoAccessToCoursersNotAssigned() throws UnsupportedEncodingException {
-//        // ARRANGE
-//        goHome();
-//
-//        // ACT
-//        goTo(getBaseUrl() + "courses?name=" + URLEncoder.encode(COURSE_NAME_I_CANNOT_ACCESS, "utf-8"));
-//
-//        // ASSERT
-//        assertThat($("h1").first().text()).isEqualTo("You do not have permissions to view this course.");
-//    }
-//
-//    private void goHome() {
-//        goTo(getBaseUrl());
-//
-//        if (!authenticated()) {
-//            loginViaOkta();
-//        }
-//    }
-//
-//    private boolean authenticated() {
-//        return !url().contains("/login");
-//    }
-//
-//    private void loginViaOkta() {
-//        $("#okta-signin-username").fill().withText(username);
-//        $("#okta-signin-password").fill().withText(password);
-//        $("#okta-signin-submit").submit();
-//        WebDriverWait wait = new WebDriverWait(driver, 10);
-//        wait.until((fluentControl) -> driver.getCurrentUrl().contains("localhost"));
-//    }
-//}
+@SpringBootTest(
+        webEnvironment = DEFINED_PORT,
+        properties = {
+                "credentials.username=foo",
+                "credentials.password=bar"
+        }
+)
+@RunWith(SpringJUnit4ClassRunner.class)
+@ActiveProfiles("cloud")
+@Ignore("Currently failing, investigation in progress")
```

Comment:
Those tests are not yet fixed and I'll have slight problems approving this pull request.

Reply:
Once #17 is merged, this problem will be solved.

Reply:
I rebased on master.